### DISCUSSION TITLE
Install on Android 4.1?

### DISCUSSION THREAD

**Discussion by Brexei** [feelfreelinux/octo4a#116] [Q&A]
Thank you for this amazing project! It was just what i hoped for, and now i found it!!!

I am trying to install octo4a (latest version) in my old samsung S3 mini which runs Android 4.1 and it says: Problem parsing the package
Is there a way to install octo4a in android 4.1? Unfortunately it cannot be upgraded to a newer version...

**Comment by ledsandchips** [comment]
Hey! Same here...
Did you make it work on your S3 Mini?

**Reply by Brexei** [reply]
No, i did not find a way to do it.
But i managed to install a custom rom (Cyanogen mod, Android 4.4) in my phone and then it worked!

**Comment by ledsandchips** [comment]
Hey!
Thanks for the reply.
Could you link me to the Cyanogen mod you used?
Cheers!

**Comment by Brexei** [comment]
You can download it from this page:
http://novafusion.pl/downloads/
It also has other versions, line 6.0 or even 7!
Install instructions:
https://howandroidhelp.com/how-to-download-and-install-android-4-4-kitkat-on-the-samsung-galaxy-s3-mini/
You can also search google for upgrading S3 mini to android 4.4
You need to install TWRP first and then boot into it. From there you can wipe all caches an system and then install the cyanogen mod.
You need to run windows 10 in your pc or else you might have problems with phone connections! It occurred to me and i had no idea how to solve it.
Good luck!

**Comment by ledsandchips** [comment]
THANK YOU!
